powerline/listers/i3wm.py

```diff
@@ -0,0 +1,81 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import re
+
+from powerline.theme import requires_segment_info
+from powerline.lib.shell import run_cmd
+
+
+conn = None
+
+
+def update_segment_info(segment_info, **kwargs):
+	copy = segment_info.copy()
+	for key in kwargs:
+		copy[key] = kwargs[key]
+	return copy
+
+
+@requires_segment_info
+def output_lister(pl, segment_info):
+	'''List all outputs in segment_info format'''
+
+	regex = re.compile('^([0-9A-Za-z-]+) connected')
```

Comment:
not moving to another file though because this regex doesn't match the other (size/position) info and I can't use the "full" regex with `.findall`